.gitignore

```diff
@@ -3,3 +3,4 @@
 *.pyc
 *.vscode
 .env
+*memory
```

Comment:
Ah I actually made a change to rename the in memory db to something that ends in `.db` so that it's gitignored. We probably won't need this

Reply:
Oh I like that better! I'll remove this